meta-balena-common/recipes-support/resin-init/resin-init-flasher/resin-init-flasher

```diff
@@ -244,8 +320,19 @@ done
 
 # Copy json configuration file from external (flasher) to the internal (booting) device
 cp -rvf "$CONFIG_PATH" "$INTERNAL_DEVICE_BOOT_PART_MOUNTPOINT"
+
+if [ "$LUKS" = "1" ]; then
+    umount "$EFI_MOUNT_DIR"
+fi
+
 umount $INTERNAL_DEVICE_BOOT_PART_MOUNTPOINT
 
+if [ "$LUKS" = "1" ]; then
+    for PART_ID in 2 3 4 5 6; do
```

Comment:
Same as before about calculating the number of partitions present.

Reply:
Changed